apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-examples/platform-file-uploader-reactive-example.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, NgForm, ValidatorFn, Validators } from '@angular/forms';
+
+import { FileUploaderInvalidChangeEvent, FileUploaderSelectionChangeEvent } from '@fundamental-ngx/platform';
+
+@Component({
+    selector: 'fdp-platform-file-uploader-reactive-example',
+    templateUrl: './platform-file-uploader-reactive-example.component.html',
+    styleUrls: ['platform-file-uploader-reactive-example.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformFileUploaderReactiveExampleComponent {
+    files: File[];
+    invalidFiles: File[];
+    customForm: FormGroup;
```

Comment:
```suggestion
    customForm: FormGroup = new FormGroup({});
    requiredDateValidator: ValidatorFn[] = [Validators.required];
```
and remove the constructor